sparkles/tests/test_checks.py

```diff
@@ -278,18 +278,18 @@ def test_guide_edge_check():
     col_lim = -(CCD['col_max'] - CCD['col_pad'] - CCD['window_pad'] - dither / 5)
 
     # Set positions just below or above CCD['guide_extra_pad'] in row / col
-    stars.add_fake_star(id=101, mag=8, row=row_lim - 2.9, col=0)
-    stars.add_fake_star(id=102, mag=8, row=row_lim - 3.1, col=0)
-    stars.add_fake_star(id=103, mag=8, row=row_lim - 5.1, col=0)
-    stars.add_fake_star(id=104, mag=8, row=0, col=col_lim + 2.9)
-    stars.add_fake_star(id=105, mag=8, row=0, col=col_lim + 3.1)
-    stars.add_fake_star(id=106, mag=8, row=0, col=col_lim + 5.1)
+    stars.add_fake_star(id=1, mag=8, row=row_lim - 2.9, col=0)
```

Comment:
`add_fake_constellation` below was actually adding stars starting from id=106, so there were two id=106 stars. I'm not sure why this was working previously, but the fix is just to not use stars with those ids here.

Reply:
I thought that was a weakness in `add_fake_constellation`, as it seems like it should just not add ids that are already in the field.  But given it is just a test routine, fine with the strategy to just manually add ids that don't conflict.

Reply:
Yes, that's all true.